### PR TITLE
[10.0][ADD] pos_disable_payment: new option 'Allow refunds'

### PR DESCRIPTION
**Pull request by ilmir-k**
[ADD] pos_disable_payment: new option 'Allow refunds'

### CODE REVIEW

**Comment by ilmir-k** [comment]
@yelizariev review it please

**Comment by yelizariev** [comment]
Basically, you need to add test for new feature:

* disable refunds
* load POS
* check that Refund button is not visible

**Comment by yelizariev** [comment]
Tests don't work.
Also, there there is not check that Refund button is not visible

**Comment by yelizariev** [comment]
You can update settings via python, no need tour for this

**Comment by yelizariev** [comment]
.numpad-minus[style="visibility: hidden;"]

File: pos_disable_payment/static/src/js/tour.js [hunk 1]
```diff
@@ -45,6 +46,16 @@ odoo.define('pos_disable_payment.tour', function(require) {
         trigger: ".o_pos_kanban button.oe_kanban_action_button",
         content: _t("<p>Click to start the point of sale interface. It <b>runs on tablets</b>, laptops, or industrial hardware.</p><p>Once the session launched, the system continues to run without an internet connection.</p>"),
         position: "bottom"
-    }]);
+    }, {
+        trigger: '.product-list .product',
```
Comment (yelizariev) [inline pos_disable_payment/static/src/js/tour.js]: This step cannot be done when pos_restaurant is installed. You need to add extra step which works both with restaurant and without one, e.g.
```

{
        content: "Swicth to table or make dummy action",
        trigger: '.table, .neworder-button:not(.oe_invisible .neworder-button)',
    }


```

**Review by yelizariev — CHANGES_REQUESTED** [review]
Tests are not passed in travis

**Comment by yelizariev** [comment]
Fix lints
don't override commits and add new ones intead

**Comment by yelizariev** [comment]
tests are not passed

File: pos_disable_payment/static/src/js/tour.js [hunk 2]
```diff
@@ -17,11 +47,19 @@ odoo.define('pos_disable_payment.tour', function(require) {
...
+        trigger: '.table:not(.oe_invisible .neworder-button), .order-button.selected',
+        position: "bottom"
+    }, {
+        trigger: '.product-list .product',
+        content: _t("Add the first product"),
+        position: "top"
+    }, {
+        content: _t("Hidden"),
+        trigger: '.pads .numpad-minus[style="visibility: hidden;"]',
+        position: "bottom"
```
Comment (yelizariev) [inline pos_disable_payment/static/src/js/tour.js]:
```
       content: _t("Hidden"),
        trigger: '.pads',
       run: function(){
// check that the button is hidden
}
```

**Review by yelizariev — CHANGES_REQUESTED** [review]

**Review by yelizariev — CHANGES_REQUESTED** [review]
remove ``_t`` from test tour -- we don't want to see those phrases in translations files

**Comment by ilmir-k** [comment]
it seems that travis has broken

**Comment by yelizariev** [comment]
2017-10-31 09:50:10,462 5497 ERROR openerp_test odoo.tests.common: phantomjs: error The button is not hidden

**Review by yelizariev — APPROVED** [review]

**Pull request merged**